Assets/General/Scripts/Managers/UniversalManagers/MainUniversalManagers/AudioManagment/StoryBeatAudioManager.cs

```diff
@@ -0,0 +1,27 @@
+using FMOD.Studio;
+using FMODUnity;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AudioManagerTester : MonoBehaviour
+{
+    [SerializeField] public EventReference[] OneshotSounds;
+    [SerializeField] public EventReference[] AmbientSounds;
+
+    [SerializeField] AudioManager Manager;
```

Comment:
add private specifier and update naming to the proper convention `_audioManager`